extensions/map-layers-formats/src/ArcGisFeature/ArcGisSymbologyRenderer.ts

```diff
@@ -87,16 +123,21 @@ export class ArcGisSimpleSymbologyRenderer  extends ArcGisSymbologyRenderer {
     let sls: EsriSLS | undefined;
     if (this._symbol.type === "esriSFS") {
       const sfs = this._symbol as EsriSFS;
-      if (sfs.outline && sfs.outline.style === "esriSLSSolid") {
```

Comment:
You can use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to make this a bit more concise.

Reply:
I agree, but you are are looking at removed code.